languages/purescript/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,59 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasagna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `expectedMinutesInOven` function that does not take any input and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
+
+```purescript
+-- Defining the function
+expectedMinutesInOven :: Int
+expectedMinutesInOven = -- Implement this function
```

Comment:
In general, the tasks in the `instructions.md` won't contain the definition of the function, as the stub file will already have that definition. Even in this case, where I argue later on that there should not be any function stubs, I would argue for removing this section as I feel it gives away too much of the solution to students. The same comment also applies to the other tasks.